app/models/contact.rb

```diff
@@ -2,8 +2,6 @@
 # these to receive messages from visitors.
 
 class Contact < ActiveRecord::Base
```

Comment:
Right, I was wondering about that as well, but we don't use it anywhere and also do not have the database table.